templates/openshift/common/dc/redirect.yml.j2

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
```

Comment:
I was expecting a "redirect" directory at the same level as "edxapp" and "richie" but the svc/dc would still be nginx:

 templates/openshift/redirect/dc/nginx.yml.j2
 templates/openshift/redirect/svc/nginx.yml.j2

WDYT?

Reply:
Totally right. Let's do this.

Reply:
It's a very good idea and close to the naming convention I propose.